dash/test/pattern/VarArgsPatternTest.cc

```diff
@@ -0,0 +1,48 @@
+#include "VarArgsPatternTest.h"
+#include "../TestBase.h"
+
+#include <gtest/gtest.h>
+
+#include <dash/Types.h>
+#include <dash/Distribution.h>
+#include <dash/TeamSpec.h>
+#include <dash/Matrix.h>
+
+#define NX 3
+#define NY 5
+
+
+TEST_F(VarArgsPatternTest, SimpleConstructorTest)
+{
+  size_t size = dash::size();
+  //  Matrix uses TilePattern by default
+  {
+    dash::Matrix<int, 2> mat(size*NX, size*NY);
+    ASSERT_GT_U(mat.local_size(), 0);
+  }
+  {
+    dash::Matrix<int, 2> mat(size*NX, size*NY,
+                             dash::BLOCKED, dash::BLOCKCYCLIC(5));
```

Comment:
Is the use of `5` instead of `NY` here on purpose?
I see the test break if NY is too low.